<!-- <?xml version="1.0" encoding="utf-8"?> -->

<launch>

    <!-- Pointcloud and image topics that should be subscribed -->
    <arg name="sub_topic_cones" default="/perception/lidar_cone_detection/cones"/>
    <arg name="sub_topic_img" default="/rgb"/>

    <!-- IDs of lidar and camera frames -->
    <arg name="frame_id_lidar" default="Lidar"/>
    <arg name="frame_id_cam" default="Camera"/>

    <arg name="get_auto_tf" default="true"/>

    <node pkg="lidar_proposal_image_classification" type="lidar_proposal_image_classification" name="pointcloud_projection_node" output="screen">
        <rosparam command="load" file="$(find lidar_proposal_image_classification)/config/camera_parameters_sim.yaml"/>

        <remap from="~sub_topic_img" to="$(arg sub_topic_img)" />
        <remap from="~sub_topic_cones" to="$(arg sub_topic_cones)" />

        <param name="frame_id_lidar" value="$(arg frame_id_lidar)"/>
        <param name="frame_id_cam" value="$(arg frame_id_cam)"/>
        <param name="get_auto_tf" value="$(arg get_auto_tf)"/>

        <!-- <param name="draw_depth" value="$(arg draw_depth)"/> -->
    </node>

</launch>